app/assets/stylesheets/kudos/components/_give-kudo.scss

```diff
@@ -3,8 +3,105 @@
   @extend .container;
   margin-bottom: $spacer;
   width: $give-kudo-width;
+  font-family: $font-gotham;
+  font-weight: 500;
+  text-align: center;
+}
+
+.give-kudo__modal {
+  text-align: left;
+  position: fixed;
+  z-index: 99 !important;
+  background: white;
+  border-radius: 4px;
+  padding: 35px 30px 35px 30px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.styled-kudo-button {
+  font-weight: 500;
+  font-size: 14pt;
+  color: $color-factual-orange;
+  background: transparent;
+  border-radius: 4px;
+  box-shadow:  $button-shadow;
+  border: none;
+  padding-right: 20px;
+  padding-left: 20px;
+
+  .fist-left {
+    padding-right: 5px;
+    transition: padding-right .1s linear 0s;
+  }
+
+  .fist-right {
+    padding-left: 5px;
+    padding-right: 15px;
+    transition: padding-left .1s linear 0s;
+  }
+}
+
+.styled-kudo-button:hover {
+  color: $hover-orange;
+  position: relative;
+  top: -1px;
+
+  .fist-left {
+    padding-right: 0;
+  }
+
+  .fist-right {
+    padding-left: 0;
+  }
+}
```

Comment:
Can you also add an `:active` state to these buttons?
There should be examples on the live demo.